### PR TITLE
keys utils testing

### PR DESCRIPTION
**Pull request by andrewkryshtal**

### CODE REVIEW

**Review by keshan3262 — APPROVED** [review]

File: src/utils/keys.utils.spec.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { mockAccountCredentials } from '../mocks/account-credentials.mock';
+import { getDerivationPath, seedToHDPrivateKey, getPublicKeyAndHash$, generateSeed } from './keys.util';
+import { rxJsTestingHelper } from './testing.utis';
+
+it('seedToPrivateKey should return derivation path, passing account index', () => {
+  expect(getDerivationPath(1)).toEqual(mockAccountCredentials.derivationPath);
```
Comment (unixvb) [inline src/utils/keys.utils.spec.ts:6]: `seedToPrivateKey` name & tested method `getDerivationPath` names missmatch

File: src/utils/keys.utils.spec.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+});
+
+it('seedToHDPrivateKey should return private key, passing seed and derivation path', () => {
```
Comment (unixvb) [inline src/utils/keys.utils.spec.ts:9]: add test for the case with empty derivation path

File: src/utils/keys.utils.spec.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { getDerivationPath, seedToHDPrivateKey, getPublicKeyAndHash$, generateSeed } from './keys.util';
+import { rxJsTestingHelper } from './testing.utis';
+
+it('getDerivationPath should return derivation path, passing account index', () => {
+  expect(getDerivationPath(1)).toEqual(mockAccountCredentials.derivationPath);
+});
+
+it('seedToHDPrivateKey should return private key, passing seed and derivation path', () => {
+  expect(
+    seedToHDPrivateKey(new Buffer(mockAccountCredentials.seedPhrase), mockAccountCredentials.derivationPath)
```
Comment (unixvb) [inline src/utils/keys.utils.spec.ts:11]: where is the test for empty derivation path?

**Review by unixvb — APPROVED** [review]
